### ISSUE TITLE
tf - keep_prob参数未起效

### ISSUE THREAD

**Issue by bifeng** [CLUEbenchmark/CLUENER2020#55]
`output_layer = tf.nn.dropout(output_layer, keep_prob=0.9)`
应为
`output_layer= tf.nn.dropout(output_layer, keep_prob=keep_prob)`
